spring-cloud-kubernetes-config/pom.xml

```diff
@@ -76,6 +76,12 @@
 			<artifactId>spock-spring</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>rest-assured</artifactId>
+			<version>3.0.2</version>
```

Comment:
I mean adding a property like 

    <restassured.version>3.0.2</restassured.version>

either on the top of the current pom, or maybe in the parent (if we are using restassured in more than one places).